core/src/apps/wallet/sign_tx/signing.py

```diff
@@ -342,7 +342,7 @@ class SigningError(ValueError):
                     addresses.ecdsa_hash_pubkey(key_sign_pub, coin)
                 )
             else:
-                raise ValueError("Unknown input script type")
+                raise SigningError("Unknown input script type")
```

Comment:
we are checking protobuf enum values so this should never happen and the exception should match that

Reply:
NO ignore me I'm wrong.

more precisely, the message is wrong, we seem to only support a subset of input scripts, so the message should say "unsupported input script type"

Reply:
Correct. I will alter the message.